artifacts/scripts/util.sh

```diff
@@ -921,7 +921,7 @@ checkout-deps-to-kube-commit() {
             git checkout -q "${dep_commit}"
 
             echo "Downloading go mod dependencies..."
-            GO111MODULE=on GOPROXY=https://proxy.golang.org go mod download
+            GO111MODULE=on GOPRIVATE="k8s.io/*" GOPROXY=https://proxy.golang.org go mod download
```

Comment:
this should come from the config, or be the source repo org, not hardcoded.

Reply:
Ah, right. Will update.